root/components/common-macros.tt

```diff
@@ -1054,3 +1054,5 @@ END -%]
     </div>
   [%- END -%]
 [%~ END ~%]
+
+[%~ MACRO non_empty(text) (text != '' && text.defined) ~%]
```

Comment:
The syntax is wrong here. :) You're missing at least BLOCK, some semicolons, and END. See for example https://github.com/stasszczesniak/musicbrainz-server/blob/master/root/components/common-macros.tt#L7